content/events/16-17/spelenmetfpgas.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Spelen met FPGAs
+description: Sigasi geeft een introductie tot FPGAs
+time: 27-04-2017
```

Comment:
This shows as midnight on the event page:

<img width="580" alt="screenshot 2017-04-13 15 51 35" src="https://cloud.githubusercontent.com/assets/1470764/25007552/2cb2d388-2061-11e7-81ac-1f7f3d4e736b.png">


I guess this needs hours and minutes. A location wouldn't hurt either.